webui/react/src/shared/utils/routes.test.ts

```diff
@@ -310,7 +309,7 @@ describe('Routes Utilities', () => {
 
     it('should route to react URL', () => {
       expect(history.push).not.toHaveBeenCalled();
-      routes.routeToReactUrl(paths.cluster());
+      routes.routeToReactUrl('/clusters');
```

Comment:
suggestion: ensure this paths and what you test for (a few lines below) don't go out of sync using a const var